test/test.jwt.ts

```diff
@@ -73,9 +73,12 @@ describe('Initial credentials', () => {
 describe('JWT auth client', () => {
   describe('.authorize', () => {
     it('should get an initial access token', done => {
-      const jwt = new JWT(
-          'foo@serviceaccount.com', PEM_PATH, null,
-          ['http://bar', 'http://foo'], 'bar@subjectaccount.com');
+      const jwt = new JWT({
```

Comment:
As far as I can tell, all the constructions in the tests are using the new options object form. Please add a test that makes sure that the other form of the `emailOrOptions is object` branch is covered.

Let me know if this is already covered and I missed it.

Reply:
Updated the first test to make sure that the non-object form is covered.